project/AppDependencies.scala

```diff
@@ -2,17 +2,19 @@ import sbt._
 
 object AppDependencies {
 
+  val MONGO_VERSION = "0.74.0"
```

Comment:
Make private final and I think name it MongoVersion ie camelcase rather than underscores